README.md

```diff
@@ -44,6 +46,28 @@ $ make
 ### Using the ISO
 - The ISO can be used with [minishift](https://github.com/minishift/minishift) with the help of `--iso-url` i.e. `--iso-url=file:///$PATHTOISO`.
 
+
+<a name="releasing-the-b2d-iso"></a>
+## Releasing the minishift-b2d ISO
+
+<a name="release-prerequisites"></a>
+### Prerequisites
+
+- Installed [Go](https://en.wikipedia.org/wiki/Go_(programming_language)) with `GOPATH` properly set.
+- A running docker daemon.
+
+<a name="releasing-the-iso"></a>
+### Releasing the ISO
```

Comment:
I'll put the releasing part after testing. First you build, then you test and finally you release. I think these are the logical steps.